lib/keystores.js

```diff
@@ -198,14 +229,25 @@ api.get = async ({id} = {}) => {
  * @param {string} options.meterId - The ID of the meter to get.
  * @param {object} options.moduleManager - The KMS module manager to use.
  * @param {AbortSignal} [options.signal] - An abort signal to check.
+ * @param {boolean} [options.explain=false] - An optional explain boolean.
  *
- * @returns {Promise<object>} The storage usage for the meter.
+ * @returns {Promise<object> | ExplainObject} The storage usage for the meter
+ *   or an ExplainObject if `explain=true`.
  */
-api.getStorageUsage = async ({meterId, moduleManager, signal} = {}) => {
+api.getStorageUsage = async ({
+  meterId, moduleManager, signal, explain = false
+} = {}) => {
+  const collection = database.collections['kms-keystore'];
+  const query = {'config.meterId': meterId};
+  const projection = {_id: 0, config: 1};
+
+  if(explain) {
+    const cursor = await collection.find(query, {projection});
+    return cursor.explain('executionStats');
+  }
+
```

Comment:
I suggest all this gets moved back to where it was before -- see my next comment below.